mediascrapers.py

```diff
@@ -136,24 +142,30 @@ def scrape(self, url):
         media_urls = []
         soup = bs(source, 'html.parser')
         title = soup.find('title').text
-        for link in soup.find_all('a', href=True):
-            if is_media(link['href']):
-                media_urls.append(link['href'])
-            if is_media(link.text):
-                media_urls.append(link.text)
+        all_links=[]
+#         for link in soup.find_all('a', href=True):
+#             if is_media(link['href']):
+#                 media_urls.append(link['href'])
+#             if is_media(link.text):
+#                 media_urls.append(link.text)
+#             all_links.append(link['href'])
         for image in soup.find_all('img', src=True):
             if is_media(image['src']):
                 media_urls.append(image['src'])
-        for video in soup.find_all('video', src=True):
-            if is_media(video['src']):
-                media_urls.append(video['src'])
+            all_links.append(image['src'])
+
+#         for video in soup.find_all('video', src=True):
+#             if is_media(video['src']):
+#                 media_urls.append(video['src'])
+#         # print(all_links)
+
 
         if self._debug:
             print(media_urls)
 
         # tasks = [(complete_url(media_url, self._driver.current_url), title, None) for media_url in media_urls]
         tasks = [(complete_url(media_url, self._driver.current_url),
-                  self._driver.current_url.replace('/','_'), None) for media_url in media_urls]
+                  self._driver.current_url[self._driver.current_url.find('status')+7:], None) for media_url in media_urls]
```

Comment:
Are sure tweetID is always 7 digits long ?

Reply:
@hotessy we are not worried about length of tweetId, since a link is 'https://twitter.com/fs0c131y/status/1286506918471700481'  like this we are just moving 7 spaces ahead of status, since 'status/' is 7 char long.